### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by fabriziovigevani**
- Fixes footer padding issue.
- Fixes suggested copies at #72 
- Fixes welcome banner width on mobile
- Updates Refuse to vote  => Abstain from voting

### CODE REVIEW

File: src/components/Dashboard/Balance.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { useCourtConfig } from '../../providers/CourtConfig'
 import { useHNYAmountToUsd } from '../../hooks/useTokenAmountToUsd'
 
 import { PCT_BASE } from '../../utils/dispute-utils'
-import { bigNum, formatTokenAmount, formatUnits } from '../../lib/math-utils'
+import { formatTokenAmount, formatUnits } from '../../lib/math-utils'
```
Comment (rperez89) [inline src/components/Dashboard/Balance.js:13]: lets order this import

File: src/components/Disputes/actions/DisputeVoting.js [hunk 2]
```diff
@@ -94,7 +94,7 @@ function VotingActions({ canJurorVote, onRequestCommit }) {
           onClick={() => onRequestCommit(VOTE_OPTION_REFUSE)}
           width={buttonWidth}
         >
-          Refuse to vote
+          Abstain from voting
         </VotingButton>
       </div>
       <RefuseToVoteHint compactMode={compactMode} width={buttonWidth} />
```
Comment (rperez89) [inline src/components/Disputes/actions/DisputeVoting.js:100]: maybe if is used only here we can change this component name to `AbstainToVoteHint` ?

File: src/components/MainView.js [hunk 3]
```diff
@@ -119,20 +119,15 @@ function MainView({ children }) {
...
                 `}
               >
                 Supporting services provided by{' '}
-                <Link href="https://aragon.one">1Hive</Link>. Read 1Hive
-                Celeste's{' '}
-                <Link href="https://anj.aragon.org/legal/terms-general.pdf">
-                  Terms of Service
-                </Link>
-                .
+                <Link href="https://wiki.1hive.org">1Hive</Link>.
```
Comment (rperez89) [inline src/components/MainView.js:130]: should we re direct to the wiki here? or the discord?

**Review by rperez89 — COMMENTED** [review]
Great fabri some minor comments

File: src/components/MainView.js (hunk 3 above)
Comment (fabriziovigevani) [reply src/components/MainView.js:130]: @willjgriff suggested to direct to the wiki. I agree it should be the wiki as well. 
There's a Discord link ion the wiki if users are looking to join Discord

**Pull request merged**